api-connection.html

```diff
@@ -100,7 +100,203 @@ <h2 id="function_newConnection">new Connection(config)</h2>
+# This could be one of the provide URIs within this link 
+# [https://docs.microsoft.com/en-us/azure/active-directory/
+# managed-identities-azure-resources/
+# services-support-managed-identities#
+# azure-services-that-support-azure-ad-authentication]
+# or any other resource URI.
+$resourceURI = "https://database.windows.net/"
+# IDENTITY_ENDPOINT/MSI_ENDPOINT & IDENTITY_HEADER/MSI_SECRET will be 
+# pick up from the app service's environment variables.
+$tokenAuthURI = $env:IDENTITY_ENDPOINT  `
++ "?resource=$resourceURI&api-version=$apiVersion"
+$tokenResponse = Invoke-RestMethod `
+-Method Get `
+-Headers @{"X-IDENTITY-HEADER"="$env:IDENTITY_HEADER"} `
+-Uri $tokenAuthURI
+$accessToken = $tokenResponse.access_token   
+{% endhighlight %}
+              <h4>A PowerShell script for retrieving the token within a msi-vm:</h4>
+              Here is a <a href="https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-powershell" target="_blank"><i><b>link</b></i></a> for microsoft side doc for retrive the token.</h4>
+{% highlight powershell %}
+# The default endpoint
+$endPoint = "http://169.254.169.254/metadata/identity/oauth2/token"
+# The corresponding api-version for the endpoint
+$apiVersion = "2018-02-01"
+# The Azure AD resource URI of the resource for which a token should be obtained.
+# This could be one of the provide URIs within this link
+# [https://docs.microsoft.com/en-us/azure/active-directory/
+# managed-identities-azure-resources/
+# services-support-managed-identities#
+# azure-services-that-support-azure-ad-authentication]
+# or any other resource URI.
+$resourceURI = "https%3A%2F%2Fmanagement.azure.com%2F"
+$tokenAuthURI = $endPoint + "?resource=$resourceURI&api-version=$apiVersion"
+$tokenResponse = Invoke-RestMethod `
+-Method Get `
+-Headers @{Metadata="true"} `
+-Uri $tokenAuthURI                              
+$access_token = $tokenResponse.access_token
+echo "The managed identities for Azure resources access token is $access_token"             
+{% endhighlight %}
+              <h4>A JavaScript for retrieve the token using 'adal-node':</h4>
+              Here is a <a href="https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-powershell" target="_blank"><i><b>link</b></i></a> for microsoft side doc for retrive the token.</h4>
+{% highlight javascript %}
+const { AuthenticationContext } = require('adal-node');
+let generateToken = () => {
+  // The Azure AD resource URI of the resource for which a token should be obtained.
+  // This could be one of the provide URIs within this link
+  // [https://docs.microsoft.com/en-us/azure/active-directory/
+  // managed-identities-azure-resources/
+  // services-support-managed-identities
+  // #azure-services-that-support-azure-ad-authentication]
+  // or any other resource URI.
```

Comment:
```suggestion
  /* The Azure AD resource URI of the resource for which a token should be obtained.
  This could be one of the provide URIs within this link
  [https://docs.microsoft.com/en-us/azure/active-directory/
  managed-identities-azure-resources/
  services-support-managed-identities
  #azure-services-that-support-azure-ad-authentication]
  or any other resource URI. */
```
Makes URL copy and paste-able